src/Microsoft.ML.CodeGen/Templates/Console/PredictProgram.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.ML.CLI.Templates.Console
     using System.Text;
     using System.Text.RegularExpressions;
     using System.Collections.Generic;
-    using Microsoft.ML.CLI.Utilities;
+    using Microsoft.ML.CodeGen.Utilities;
```

Comment:
Something seems off here. There is no corresponding change in the tt files.

Reply:
General rule of thumb. Do not explicitly modify any .cs files associated with templates. They should be always auto-generated.

Reply:
Or even better. have the files generated on build of the project